modules/utility/init.zsh

```diff
@@ -6,7 +6,8 @@
 # Colours
 #
 
-if (( ${terminfo[colors]} >= 8 )); then
```

Comment:
@drm00 nope, this was all you, just recommending a small syntax change. If you update your commit, I'll merge this into master. Thanks!